test/t8n/t8n.cpp

```diff
@@ -111,6 +111,26 @@ int main(int argc, const char* argv[])
 
                     const auto computed_tx_hash = keccak256(rlp::encode(tx));
 
+                    auto sender_acc_ptr = state.find(tx.sender);
+                    if (sender_acc_ptr == nullptr)
+                    {
+                        // It's possible to send tx from non-existent account before London fork.
+                        // TODO: This should be handled in `validate_transaction` but it requires
```

Comment:
I'm not sure. We should not assume the sender account exist.

Try changing in `transition()`:
```cpp
auto& sender_acc = state.get(tx.sender);
```
to
```cpp
auto& sender_acc = state.get_or_insert(tx.sender);
```